pkgs/development/libraries/ffmpeg/generic.nix

```
@@ -121,7 +123,7 @@ stdenv.mkDerivation rec {
       (ifMinVer "0.6" "--enable-avdevice")
       "--enable-avfilter"
       (ifMinVer "0.6" "--enable-avformat")
-      (ifMinVer "1.0" "--enable-avresample")
+      (ifMinVer "1.0" (ifMaxVer "4.4" "--enable-avresample"))
```

I think because of that we need to target staging.

---

Ah, did I mess that up? 5.0 doesn't have avresample anymore.

---

if this an exclusive bounds check, then you probably want 
```suggestion
      (ifMinVer "1.0" (ifMaxVer "5.0" "--enable-avresample"))
```

---

If ofborg reports less rebuilds back when switch back to master and then squash merge.

---

That didn't work...

```
configure flags: --disable-static --prefix=/nix/store/xzgqkbn1vlyh0vag6czbkhwvvmafhk6f-ffmpeg-5.0 --arch=x86_64 --target_os=linux --enable-gpl --enable-version3 --enable-shared --enable-pic --enable-libsrt --enable-runtime-cpudetect --enable-hardcoded-tables --enable-pthreads --disable-w32threads --disable-os2threads --enable-network --enable-pixelutils --enable-ffmpeg --disable-ffplay --enable-ffprobe --enable-avcodec --enable-avdevice --enable-avfilter --enable-avformat --enable-avresample --enable-avutil --enable-postproc --enable-swresample --enable-swscale --disable-doc --enable-libass --enable-bzlib --enable-gnutls --enable-fontconfig --enable-libfreetype --enable-libmp3lame --enable-iconv --enable-libtheora --enable-libssh --enable-vaapi --enable-libdrm --enable-vdpau --enable-libvorbis --enable-libvpx --enable-lzma --disable-opengl --disable-libmfx --disable-libaom --enable-libpulse --enable-sdl2 --enable-libsoxr --enable-libx264 --enable-libxvid --enable-libzimg --enable-zlib --enable-libopus --enable-libspeex --enable-libx265 --enable-libdav1d --disable-debug --enable-optimizations --disable-extra-warnings --disable-stripping
Unknown option "--enable-avresample".
See ./configure --help for available options.
builder for '/nix/store/jd9xs7h4q1ak0j5kxraq9difbjxyna8x-ffmpeg-5.0.drv' failed with exit code 1
error: build of '/nix/store/jd9xs7h4q1ak0j5kxraq9difbjxyna8x-ffmpeg-5.0.drv' failed
```